idaes/core/surrogate/keras/sparsify_prune_example.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
Is this the right place for this file? Most example notebooks belong in the separate `examples-pse` repository.

Reply:
Yes the location of this file was something Michael and Carl talked about during development of these changes.  We decided to put it here for now but it could always be moved!